frontend/src/modules/Shared/Shares/ShareEditForm.js

```diff
@@ -275,6 +276,10 @@ export const ShareEditForm = (props) => {
 
     if (onApply) {
       onApply();
+      const targetPath = `/console/shares/${share.shareUri}`;
+      if (location.pathname !== targetPath) {
+        navigate(targetPath);
+      }
```

Comment:
can we move L279-282 outside of the `if (onApply) { ...}` block

even if there is not a defined function `onApply` (coming from the parent component) I imagine we would want to redirect the user to the share view

Reply:
Okay that makes sense.

Reply:
@noah-paige Please review